### ISSUE TITLE
Batch Norm Concerns

### ISSUE THREAD

**Issue by CJMenart** [CJMenart/Direct-Prior-Prediction-CNN#8]
Add batch norm to added layers, make sure they update properly.

Also may wish to make sure that tf.slim networks are actually adding their batch-norm operations to tf.UPDATES when appropriate, because I've been catching it empty...
